src/index.ts

```diff
@@ -108,6 +113,24 @@ export class Client {
     return this.request<IEmailSetResponse>('Email/set', args);
   }
 
+  public emailSubmission_get(
+    args: IEmailSubmissionGetArguments,
+  ): Promise<IEmailSubmissionResponse> {
+    return this.request<IEmailSubmissionResponse>('EmailSubmission/get', args);
+  }
+
+  public emailSubmission_changes(
```

Comment:
Just to be sure you are aware of it: EmailSubmission/changes is not implemented by James